lib/SIL/IR/SILFunctionType.cpp

```diff
@@ -4233,6 +4245,7 @@ TypeConverter::getLoweredFormalTypes(SILDeclRef constant,
     bridgedResultType = resultType;
     break;
 
+  case SILFunctionTypeRepresentation::CXXMethod:
```

Comment:
In the body of the case, you could simplify the code to check if the representation is `CXXMethod` instead of doing `auto method = dyn_cast_or_null<clang::CXXMethodDecl>(constant.getDecl()->getClangDecl())`.

Reply:
We need `method`, so we'd end up casting it anyway, so I'm not sure how much better it would be. But happy to update if you'd still like me to.